src/JokeraceEligibility.sol

```diff
@@ -189,6 +206,14 @@ contract JokeraceEligibility is HatsEligibilityModule {
       revert JokeraceEligibility_TermNotCompleted();
     }
 
+    GovernorCountingSimple newContest = GovernorCountingSimple(payable(newUnderlyingContest));
```

Comment:
I suggest converting this code into an internal function and using it both here and in _setUp()